create_jenkins_job.py

```diff
@@ -191,15 +191,15 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 'cmake_build_type': 'Debug',
                 'compile_with_clang_default': 'true',
                 'time_trigger_spec': PERIODIC_JOB_SPEC,
-                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
+                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS + ',amazon-b9-ros@amazon.com',
```

Comment:
Are you sure they are comma separated? Other Jenkins jobs use a space to separate emails.

Reply:
Ah no, hazarded a guess. Fixed in 6e02283.